python/triton/runtime/jit.py

```diff
@@ -5,7 +5,9 @@
 import itertools
 import os
 import re
+import sys
 import textwrap
+import torch
```

Comment:
Having a torch dependency here is not great

Reply:
Any suggestions on how could I avoid it?

Reply:
Not an expert on python or pytorch, but ideas come to my mind: `str(type(arg))` and check it's `torch.Tensor`? `has_attr(arg, untyped_storage)` together with that? Maybe there are better ways though. Is `torch.Tensor` implementing some "interface"?